testing/grpc/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: wener <wenermail@gmail.com>
+# Maintainer: wener <wenermail@gmail.com>
+pkgname=grpc
+pkgver=1.10.0
+pkgrel=0
+pkgdesc="The C based gRPC (C++, Python, Ruby, Objective-C, PHP, C#) "
+url="https://grpc.io/"
+giturl="https://github.com/grpc/grpc"
+arch="all"
+license="Apache2"
+depends="protobuf"
+depends_dev="protobuf-dev"
+makedepends="autoconf automake libtool libstdc++ protobuf-dev c-ares-dev openssl-dev gflags-dev gtest-dev zlib-dev yaml-dev"
+subpackages="$pkgname-dev $pkgname-cli"
+_googletest_rev=ec44c6c1675c25b9827aacd08c02433cccde7780
+source="$pkgname-$pkgver.tar.gz::https://github.com/grpc/grpc/archive/v$pkgver.tar.gz
+		googletest-$_googletest_rev.tar.gz::https://github.com/google/googletest/archive/$_googletest_rev.tar.gz
+		01-chttp2-maybe-uninitialized.patch"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!check"
+
+unpack() {
+	default_unpack
+	cd "$srcdir"
+	rm -rf $pkgname-$pkgver/third_party/googletest
+	ln -sfn $PWD/googletest-$_googletest_rev/ $pkgname-$pkgver/third_party/googletest
+}
+
+build() {
+	cd "$builddir"
+	make -j$(nproc)
```

Comment:
you dont need to (and should not) set `-j$(nproc)` because we set this in `/etc/abuild.conf`:
````
export JOBS=20
export MAKEFLAGS="-j$JOBS"
````